### PR TITLE
Added JRuby 9k to teh tests, as we're going to use that in prod soon

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

**Comment by paultag** [comment]
:+1: waiting on test success

**Comment by plusjeff** [comment]
Will test locally while tests are running.

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 language: ruby
-rvm:
-    - jruby-1.7.22
+matrix:
+    include:
+        - rvm: jruby-1.7.22
+          env: JRUBY_OPTS="--2.0" CONNECT_VBMS_ENV_DIR="/" CONNECT_VBMS_ENV="test" CONNECT_VBMS_URL="https://" CONNECT_VBMS_KEYFILE="keystore.jks" CONNECT_VBMS_KEYPASS="nopass" CONNECT_VBMS_SAML="samltoken.xml"
+        - rvm: jruby-9.0.1.0
```
Comment (plusjeff) [inline .travis.yml]: Can this be bumped to `jruby-9.0.3.0`?

**Comment by plusjeff** [comment]
Tests out fine locally with JRuby 9.0.3.0. Will merge after tests complete

**Pull request merged**